src/DynamoInstall/DynamoInstall.wixproj

```diff
@@ -28,8 +28,11 @@
     </WixVariables>
   </PropertyGroup>
   <ItemGroup>
+    <Compile Include="MyMaintenanceTypeDlg.wxs" />
```

Comment:
You can rename MyMaintenanceTypeDlg to MaintenanceDialog.wxs and likewise the other one to VerificationDialog.wxs